packages/web/src/server/index.ts

```diff
@@ -130,7 +131,11 @@ export async function track(
       }
     });
 
-    await promise;
+    if (requestContext?.waitUntil) {
+      requestContext.waitUntil(promise);
```

Comment:
**praise:** <picture data-single-emoji=":dope:" title=":dope:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/dope/b5a494ec6728c48b.png" alt=":dope:" align="absmiddle"></picture> Huge thanks to the compute team!